### ISSUE TITLE
'convert-html-to-dash' is a package and cannot be directly executed

### ISSUE THREAD

**Issue by aerosadegh** [xhluca/convert-html-to-dash#1]
I want to post new Issue to your project but can't.

I got this error when try to use your module to convert an html to dash html in windows 10 (python 3.8).

```
...>python -m convert-html-to-dash -h
C:\Python38\python.exe: No module named convert-html-to-dash.__main__; 'convert-html-to-dash' is a package and cannot be directly executed
```

**Comment by firmai** [comment]
Hi @xhlulu I have been having the same problem, can I also enquire whether this is still the best way to convert a bulk of HTML code?

**Comment by tompeled** [comment]
python3 -m convert-html-to-dash -h
 No module named convert-html-to-dash.__main__; 'convert-html-to-dash' is a package and cannot be directly executed

same problem for me
any solution?

**Comment by tompeled** [comment]
@xhlulu

**Comment by xhluca** [comment]
This is not a package. There's no main file so you can't use this as a command line tool. 

You will need to run convert.py, which might not work since it's just a personal experiment and not a real project.

**Comment by xhluca** [comment]
> Hi @xhlulu I have been having the same problem, can I also enquire whether this is still the best way to convert a bulk of HTML code? 

@firmai I can't really say if this is the right way. I've tried using this script for simple use cases, which seemed to work. I then applied the script to a more complex use case, and it seems to fail. I don't remember what the html files looked like, however I would say the use of html tree parsing and dash component construction is in the right direction; more error and edge case catching could yield an improved result.
